### PR TITLE
Backport migration to python 3 to 7.x

### PR DESCRIPTION
**Pull request by jsoriano**
Backports #14798 to 7.x branch.

### CODE REVIEW

**Comment by jsoriano** [comment]
run beats-ci/package

**Comment by urso** [comment]
Jenkins did fail in x-pack/filebeat zeek tests. These are rather flaky recently in general.

**Comment by urso** [comment]
Jenkins, test this.

File: x-pack/metricbeat/module/appsearch/docker-compose.yml [hunk 1]
```diff
@@ -2,8 +2,11 @@ version: '2.3'
 
 services:
   appsearch:
+    image: docker.elastic.co/integrations-ci/beats-appsearch:${APPSEARCH_VERSION:-7.5.0}-1
     build:
       context: ./_meta
+      args:
+        APPSEARCH_VERSION: ${APPSEARCH_VERSION:-7.5.0}
```
Comment (urso) [inline x-pack/metricbeat/module/appsearch/docker-compose.yml]: We should use 7.7.0-SNAPSHOT here, so to test against current 7.x branch.

**Pull request merged**